index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/fddcccfd38.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Local TubeClone</title>
+</head>
+
+<body>
+    <!--header start-->
+    <header>
+        <nav class="flex-container">
+            <i class="fas fa-bars"></i>
+            <img class="logo" src="images/logo" alt="image of logo">
+            <form action="">
+                <input class="myInput" type="text" id="search" placeholder="Search"><i class="fas fa-search"></i>
+            </form>
+
+            <i class="fas fa-upload"></i>
+            <button class="sign" type="submit">Sign in</button>
+        </nav>
+        <article>
+            <div class="mini-banner">
+                <p><i class="far fa-user"></i></p>
+                <h6>Choose your Language.</h6>
+                <a href="#" class="learn_more">X</a>
+            </div>
+            <div class="language">
+                <p class="smaller smaller-left">You are viewing Youtube in <b>English (US)</b>. You can <a href="#">Change this preference
+                        below.</a></p> <a class="smaller smaller-right" href="#">Learn More</a>
+
+            </div>
+        </article>
+    </header>
+    <!--header ends-->
+
+    <!--main content-->
+    <div class="container">
+        <main>
+            <div class="video-container">
+                <iframe class="vid" src="https://www.youtube.com/embed/KG3phuIo_JU"></iframe>
```

Comment:
- [ ] Please check that your video can be displayed ( I would recommend you using another link)
![image](https://user-images.githubusercontent.com/53256139/91040181-685ce500-e5d3-11ea-9853-c320bde8986f.png)